scripts/fetch-yochu-assets

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
-ETCD_VERSION=v2.2.5-gs-1
+ETCD_VERSION=v3.0.3
 FLEET_VERSION=v0.11.5-gs-grpc-1
 DOCKER_VERSION=1.10.3
-YOCHU_VERSION=0.19.1
-RKT_VERSION=v1.3.0
-K8S_VERSION=v1.2.2
+YOCHU_VERSION=0.20.1
```

Comment:
There is no release of yochu 0.20.1. @hectorj2f according to the date it looks like it contains your iptables template changes. can you create the release?